_articles/graphics-switch-pop.md

```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
 Applications will use the integrated GPU unless explicitly requested to use the
 discrete GPU.
 
+Vulkan applications must be launched with the following command before the application name to be rendered on the dGPU (NVIDIA):
+
+```
+__NV_PRIME_RENDER_OFFLOAD=1 *application*
+```
+
+GLX applications much be launcher with this command before the application to be rendered on the dGPU (NVIDIA):
```

Comment:
A right click context menu item is also available in newer versions of GNOME Shell to launch applications on the dGPU. (Did it make it into 3.35.91?)

Reply:
I'll need to get a machine that can do that to test.

Reply:
@crawfxrd I just upgraded to GNOME 3.35.91 and I don't see the option in Pop.